lib/managers/webhooks.js

```diff
@@ -110,6 +110,17 @@ function validateDeliveryTimestamp(headers, maxMessageAge) {
 	return true;
 }
 
+/**
+ * Stringify JSON with escaped multibyte Unicode characters to ensure computed signatures match PHP's default behavior
+ *
+ * @param {Object} body - The parsed JSON object
+ * @returns {string} - Stringified JSON with escaped multibyte Unicode characters
+ * @private
+ */
+function jsonStringifyWithEscapedUnicode(body) {
+	return JSON.stringify(body).replace(/[\u007f-\uffff]/g, char => `\\u${`0000${char.charCodeAt(0).toString(16)}`.slice(-4)}`);
```

Comment:
Whats the logic here?

Reply:
This code snippet escapes all non-ASCII characters in their unicode escape sequence "\\u....". `charCodeAt()` returns the UTF-16 code unit from 0-65535, and `toString(16)` returns the hexademical value. So "007f"-"ffff" is numbers 127-65535, which is how the regex captures all the non-ASCII characters.

This is needed to match PHP's `json_encode` which also encodes all non-ASCII characters using "\u...." escape sequences. This stack overflow answer has a good explanation: https://stackoverflow.com/a/22745663/12092405